tools/pinning/oldest/pyproject.toml

```diff
@@ -36,84 +36,59 @@ acme = {path = "../../../acme", extras = ["test"]}
 
 # Oldest dependencies
 # We specify the oldest versions our dependencies that we're trying to keep
-# support for below. Usually these version numbers are taken from the packages
-# of our dependencies available in popular LTS Linux distros. Keeping
-# compatibility with those versions makes it much easier for OS maintainers to
-# update their Certbot packages.
+# support for below. We should only update these packages as needed to make use
+# of features in newer versions of our dependencies. Keeping compatibility with
+# older packages makes it much easier for OS maintainers to update their
+# Certbot packages if needed or desired.
 #
 # When updating these dependencies, we should ideally try to only update them
-# to the oldest version of the dependency that is found in a non-EOL'd version
-# of CentOS, Debian, or Ubuntu that has Certbot packages in their OS
-# repositories using a version of Python we support. If the distro is EOL'd or
-# using a version of Python we don't support, it can be ignored. If the
-# dependency being updated is a direct dependency of one of our own packages,
-# the minimum required version of that dependency should be updated in our
-# setup.py files as well to communicate this information to our users.
+# to the oldest version of the dependency found in CentOS/RHEL 8 + EPEL (or
```

Comment:
```suggestion
# the oldest version of the dependency found in CentOS/RHEL 8 + EPEL (or
```

Reply:
I think whether or not we should make this change depends on the outcome of https://github.com/certbot/certbot/pull/8999#discussion_r704852799.